Add support for mount options

pagdot opened the pull request:
In the config.txt all additional columns get treated forwarded as parameters

RealOrangeOne reviewed (approved):
This change looks great, although you'll need to make shellcheck happy with the lack of quotes.

pagdot commented:
Hmm.... The issue is that the missing quotes are intentional because I want it to split to multiple arguments. I'll take a look though into using a bash array instead which could be better altogether because at the moment I can't use quotes in the config.txt to escape whitespaces

pagdot commented:
Hmm.... I don't know right now an alternative way to implement proper parsing without relying on `eval`. Would it be okay for you to trick shellcheck by assigning an empty array to arr? Like with `arr=()`?

pagdot committed:
Rewrite to handle parameter expansion

Changed the line parsing to parse the line as array with `eval` to keep the parameter expansion

pagdot committed:
Add support for mount options

In the config.txt all additional columns get treated forwarded as parameters

pagdot committed:
use xargs + readarray to circumvent eval

pagdot commented:
Hi, fixed it finally after ignoring it for.... half a year.....

I've setup the build system in my own fork to auto build for new rclone version which I'll use moving forward
